src/app/app-components/courses/courses-list/courses-list.component.ts

```diff
@@ -8,31 +8,50 @@ import { Course } from 'src/app/app-models/course';
   styleUrls: ['./courses-list.component.scss']
 })
 export class CoursesListComponent implements OnInit {
+  private readonly LOREM_IPSUM = `Lorem ipsum dolor sit amet, 
+  consectetur adipiscing elit, sed do eiusmod tempor incididunt 
+  ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis 
+  nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo 
+  consequat. Duis aute irure dolor in reprehenderit in voluptate velit 
+  esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat 
+  cupidatat non proident, sunt in culpa qui officia deserunt mollit anim 
+  id est laborum`;
+
   private readonly FAKE_COURSE_1 = {
     id: 'id_1',
     title: 'Course 1',
     creationDate: new Date(100000000),
     duration: 10,
-    description: 'Course 1 description...',
+    description: 'Course 1 description... ' + this.LOREM_IPSUM,
```

Comment:
We can use the ES6 template strings here:

```typescript
    description: `Course 1 description...${this.LOREM_IPSUM}`
```

It's preferred than concatenating.

Reply:
OK